packages/client/tests/functional/chunking-query/tests.ts

```diff
@@ -255,5 +255,9 @@ testMatrix.setupTestSuite(
       // query server parameter.
       reason: 'Vitess supports at most 10k rows returned in a single query, so this test is not applicable.',
     },
+    skipDataProxy: {
+      runtimes: ['node', 'edge'],
+      reason: 'Unreachable assertions in the data proxy',
```

Comment:
What does "unreachable assertions" mean here?
What happens for Data Proxy / Mini Proxy?

Reply:
As written elsewhere, the CI fails on *Proxy, as the test execution runs "unreachable assertions" (which appear in the code as `expect(true).toBe(false)`). The `skipDataProxy` settings avoids this problem.

I'm very confused about why such gimmick is needed in the first place, though.

Reply:
I've found the problem, and I'm fixing it.

Reply:
It's probably fixed now. I've discovered that, if we run the MySQL-equivalent snippet of https://github.com/prisma/prisma/issues/21802 (`n = 32767`), we don't get an error at all (as opposed to Postgres/CockroachDB, where we get the error "too many bind variables in prepared statement"). However, if we increase `n` by 1 (`n = 32767`), we get the error "Prepared statement contains too many placeholders", which is analogous to Postgres' "too many bind variables in prepared statement"

Reply:
Now I'm investigating a potential race condition: sometimes (with Neon), we get an error message containing "bind message has 5 parameter formats but 0 parameters", although the expectation is "bind message has 32767 parameter formats but 0 parameters".

Reply:
Including MysQL definitely makes this more messy, but good learning! Seems the missing chunking behaves different there than with PostgreSQL. Good call 👍